Source/Scene/SkyAtmosphere.js

```diff
@@ -96,6 +132,12 @@ define([
         this._command.uniformMap = {
             cameraAndRadiiAndDynamicAtmosphereColor : function() {
```

Comment:
Missed this from before, but for consistency this should be `u_cameraAndRadiiAndDynamicAtmosphereColor`.